starknet_py/abi/v1/parser_transformer.py

```diff
@@ -26,20 +34,27 @@ def type(self, value):
     def type_felt(self, _value):
         return FeltType()
 
-    def type_uint(self, _value):
-        return FeltType()
+    def type_uint(self, value):
+        return UintType(int(value[0]))
 
     def type_unit(self, _value):
         return None
 
     def type_option(self, _value):
-        return Option()
+        return Option(_value[0])
```

Comment:
What do you think about naming it `OptionType`? I know it's an enum technically. 

cc @drknzz

Reply:
https://github.com/starkware-libs/cairo/blob/v1.0.0-alpha.6/corelib/src/option.cairo#L3